scripts/core/find_temporal_crf_core.m

```diff
@@ -0,0 +1,79 @@
+function [results] = find_temporal_crf_core(best_model,shuffle_model,data,stimuli, Coord_active)
```

Comment:
Confirm it works with `time_span=1`

Reply:
Also change the name to make clear it plots. Compare to `find_temporal_ens_nodes`.

Reply:
Done in 2b6e01a.
`time_span=1` runs successfully, tho could use more content testing still.